app/views/social_previews/articles/article.html.erb

```diff
@@ -1,5 +1,7 @@
 <% accent_color = Color::CompareHex.new([user_colors(@article.user)[:bg], user_colors(@article.user)[:text]]).biggest %>
-<% dark_color = Color::CompareHex.new([user_colors(@article.user)[:bg], user_colors(@article.user)[:text]]).brightness(0.8) %>
+<% dark_color = Color::CompareHex.new([user_colors(@article.user)[:bg], user_colors(@article.user)[:text]]).brightness(0.8). %>
+<% then {|color| "##{Color::Accessibility.new(color).reduce_brightness}" } %>
+
```

Comment:
Since this is solely external facing, ie social preview, I'd recommend using a default of black when the contrast is too low.

Reply:
Sure, I've made the change.